Add examples  to testing orders page (#53)
ShopBonsai/docs: docs/api/orders/orders-test-info.md

### Using Bonsai Test in Customer Shipping Info:

1. Make a call to our submit orders api using our API key.

> 2. Add "bonsai-test" to the customer's first name and last name in the shipping address.

Reviewer: ```suggestion
2. Add "bonsai" as customer's first name in the shipping address.
3. Add "test" as customer's last name in the shipping address.
```

### Using Bonsai Test in Customer Shipping Info:

1. Make a call to our submit orders api using our API key.

> 2. Add "bonsai" to the customer's first name in the shipping address.
> 3. Add "test" to the customer's last name in the shipping address.

Reviewer: I believe this is more accurate.

```suggestion
2. Set the customer's first name to "bonsai"  in the shipping address.
3. Set the customer's last name to "test"  in the shipping address.
```

> 2. Set "bonsai" to the customer's first name in the shipping address.
> 3. Set "test" to the customer's last name in the shipping address.

Reviewer: ```suggestion
2. Set "bonsai" as the customer's first name in the shipping address.
3. Set "test" as the customer's last name in the shipping address.
```
